projects/ngx-powerbi/package.json

```diff
@@ -24,8 +27,9 @@
     "reporting"
   ],
   "peerDependencies": {
-    "@angular/common": "^7.0.0",
-    "@angular/core": "^7.0.0",
-    "powerbi-client": "^2.6.5"
+    "@angular/common": "^9.0.6",
+    "@angular/core": "^9.0.6",
```

Comment:
Angular versions in main package.json are ^9.0.1 but here the versions are ^9.0.6, can you make them same? Better option would be to update angular versions to the latest one i.e. 9.1.2

Reply:
I updated to 9.1.2